### DISCUSSION TITLE
cannot have an empty name;

### DISCUSSION THREAD

**Discussion by CodePranjal** [duncanmcclean/simple-commerce#1169] [Help]
Hello
I am getting this error, not sure why. 
The command defined in "DuncanMcClean\SimpleCommerce\Console\Commands\GeneratorCommand" cannot have an empty name. 

found the file causing it:

<?php

namespace DuncanMcClean\SimpleCommerce\Console\Commands;

use Statamic\Console\Commands\GeneratorCommand as StatamicGeneratorCommand;

class GeneratorCommand extends StatamicGeneratorCommand
{
    /**
     * We need to do this ourselves so it uses the
     * Simple Commerce stub path.
     *
     * @return string
     */
    protected function getStub($stub = null)
    {
        $stub = $stub ?? $this->stub;

        return __DIR__.'/stubs/'.$stub;
    }
}

would you be able to help? thanks

**Comment by duncanmcclean** [comment]
Please update to Statamic v5.29.0 and Simple Commerce v7.5.2.
